src/server/routers/user-route.ts

```diff
@@ -28,6 +28,7 @@ userRouter.post("/create-user", async function (req:any, res:any) {
          roles:[role?._id]
          
         });
+        user.save().then(()=>res.status(200).json({data:user}))
     });
```

Comment:
```suggestion
    .catch(err => res.status(501).json(err))
    });
```
i like having the catch there .. but not madatory of course.